.github/workflows/main.yml

```
@@ -0,0 +1,13 @@
+on:
+  - push
+  - pull_request
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@v2
        with:
          fetch-depth: 0
```

I would make it fetch it the complete history or big enough number of commits that can't happen in a month because version 2 of checkout action defaults to a shallow checkout of only the last version and will potentially fail the one month old requirement.